src/toil_scripts/adam_pipeline/adam_preprocessing.py

```diff
@@ -36,11 +36,12 @@
 
 import yaml
 from toil.job import Job
```

Comment:
Currently all of the pipelines I refactored have the imports ordered like this as per PEP8 guidelines. I'm happy to change it, but it'll reduce consistency.